src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories/DdsInstrumentationContext.cs

```diff
@@ -50,13 +51,14 @@ public int ClearValidBatches(int from, int to)
             return Database.ExecuteSqlRaw(sql);
         }
 
-        public async Task<WorkflowJob> PutJob(string bNumber, bool forceRebuild, bool take, int? workflowOptions,
+        public async Task<WorkflowJob> PutJob(string identifier, bool forceRebuild, bool take, int? workflowOptions,
```

Comment:
should `string identifier` be `DdsIdentifier identifier`?

Reply:
Yes... I'll see how far that cascades if I make that change.